src/main/java/com/ookawara/book_/application/entity/Book.java

```diff
@@ -1,25 +1,24 @@
-package com.ookawara.book_.application.Entity;
+package com.ookawara.book_.application.entity;
 
 import java.time.LocalDate;
-import java.util.Date;
 
 public class Book {
-    private int id;
+    private int books_id;
     private String name;
     private LocalDate release_date;
     private boolean is_purchased;
     private int category_id;
 
-    public Book(int id, String name, LocalDate release_date, boolean is_purchased, int category_id) {
-        this.id = id;
+    public Book(int books_id, String name, LocalDate release_date, boolean is_purchased, int category_id) {
+        this.books_id = books_id;
         this.name = name;
         this.release_date = release_date;
         this.is_purchased = is_purchased;
         this.category_id = category_id;
     }
 
-    public int getId() {
-        return id;
+    public int getBooks_id() {
```

Comment:
メソッド名にアンダースコアを使うのはNG。
Javaでは小文字始まりのキャメルケースを使うのが慣習です。
```suggestion
    public int getBooksId() {
```
https://www.oracle.com/java/technologies/javase/codeconventions-namingconventions.html
> Methods should be verbs, in mixed case with the first letter lowercase, with the first letter of each internal word capitalized.

Reply:
修正しました。
1eaeddf6a492993fccdb91a9e4caafaaa982193e

Reply:
👍